### PR TITLE
added cards and adjusted pictures and worked on nav and footer

### PR DESCRIPTION
**Pull request by kenymc1**

### CODE REVIEW

File: Fan.html [hunk 1]
```diff
@@ -34,7 +34,7 @@
             <a class="nav-link" href="bio.html">Bio</a>
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="Fan.html">Fans</a>
+            <a class="nav-link" href="Fan.html">Fans Page</a>
```
Comment (ancasimon) [inline Fan.html:37]: Why did you add "Page" in the nav link name? None of the other links have the word "Page" in them...

File: Fan.html [hunk 2]
```diff
@@ -69,7 +69,7 @@
           <i class="fab fa-youtube-square"></i>
           <i class="fab fa-snapchat-square"></i>
         </div>
-        <p class="text-center">©2020 Battle Heart</p>
+        <p class="text-center">©2020 Battleheart</p>
```
Comment (ancasimon) [inline Fan.html:72]: 👍 :)

File: bio.html [hunk 3]
```diff
@@ -34,7 +34,7 @@
             <a class="nav-link" href="bio.html">Bio</a>
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="Fan.html">Fans</a>
+            <a class="nav-link" href="Fan.html">Fans Page</a>
```
Comment (ancasimon) [inline bio.html:26]: same question as above

File: dates.html [hunk 4]
```diff
@@ -33,7 +33,7 @@
             <a class="nav-link" href="bio.html">Bio</a>
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="Fan.html">Fans</a>
+            <a class="nav-link" href="Fan.html">Fans Page</a>
```
Comment (ancasimon) [inline dates.html:36]: ditto

File: discography.html [hunk 5]
```diff
@@ -34,7 +34,7 @@
             <a class="nav-link" href="bio.html">Bio</a>
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="Fan.html">Fans</a>
+            <a class="nav-link" href="Fan.html">Fans Page</a>
```
Comment (ancasimon) [inline discography.html]: ditto

**Comment by davideverett89** [comment]
👍

**Comment by davideverett89** [comment]
👍
